README.yaml

```diff
@@ -106,59 +35,152 @@ badges:
 
-# List any resources helpful for someone to get started. For example, link to the hashicorp documentation or AWS documentation.
 references:
+  - name: Glue Getting Started Guide
+    description: Guide for getting oriented with glue and spark
+    url: https://docs.aws.amazon.com/glue/latest/dg/getting-started.html
+  - name: Program AWS Glue ETL Scripts in Python
+    description: Documentation about the process of running ETL with AWS Glue and the Python programming language
+    url: https://docs.aws.amazon.com/glue/latest/dg/aws-glue-programming-python.html
 
 # Short description of this project
 description: |-
-  Short
-  description
+  Terraform modules for provisioning and managing AWS [Glue](https://docs.aws.amazon.com/glue/latest/dg/what-is-glue.html) resources. 
+  
+  The following Glue resources are supported:
+  
+    - Catalog database
+    - Catalog table
+    - Connection
+    - Crawler
+    - Job
+    - Registry
+    - Schema
+    - Trigger
+    - Workflow
+
+  Refer to [modules](modules) for more details.
+
+# How to use this module
+usage: |-
+  For an example on how to provision source and destination S3 buckets, Glue Catalog Database and Table, and a Glue Crawler that processes 
+  data in the source S3 bucket and writes the result into the destination S3 bucket, 
+  see [examples/crawler](examples/crawler).
 
-# Introduction to the project
-introduction: |-
-  This is an introduction.
+  For an example on how to provision a Glue Workflow with Glue Jobs and Glue Triggers, 
+  see [examples/complete](examples/complete).
 
-# How to use this module. Should be an easy example to copy and paste.
-usage: |-
-  For a complete example, see [examples/complete](examples/complete).
+  For automated tests of the examples using [bats](https://github.com/bats-core/bats-core) and [Terratest](https://github.com/gruntwork-io/terratest)
+  (which tests and deploys the examples on AWS), see [test](test).
 
-  For automated tests of the complete example using [bats](https://github.com/bats-core/bats-core) and [Terratest](https://github.com/gruntwork-io/terratest)
-  (which tests and deploys the example on AWS), see [test](test).
+# Example usage
+examples: |-
 
   ```hcl
-  # Create a standard label resource. See [null-label](https://github.com/cloudposse/terraform-null-label/#terraform-null-label--)
-  module "label" {
-    source  = "cloudposse/label/null"
-    # Cloud Posse recommends pinning every module to a specific version, though usually you want to use the current one
-    # version = "x.x.x"
-
-    namespace = "eg"
-    name      = "example"
+  
+  locals {
+    s3_bucket_job_source_name = module.s3_bucket_job_source.bucket_id
+    role_arn                  = module.iam_role.arn
   }
 
-  module "example" {
-    source  = "cloudposse/*****/aws"
-    # Cloud Posse recommends pinning every module to a specific version
-    # version = "x.x.x"
+  module "s3_bucket_job_source" {
+    source  = "cloudposse/s3-bucket/aws"
```

Comment:
Please omit the version here and add our common generic inline comment that we use in other module readmes